### ISSUE TITLE
Comments in column data format header

### ISSUE THREAD

**Issue by dhubber** [gandalfcode/gandalf#37]
It was suggested in the GANDALF school to add a comment character (hash I think) to allow simple plotting programmes (such as gnuplot) to ignore the header when reading in the file.  However, the column format can also be used for reading in snapshots.  Would we need to read in and strip away this hash (assuming it is there in case people load in older snapshots) or is there a simpler way to ignore it in C++?

**Comment by rbooth200** [comment]
Hhmm. So I think the best thing to do is to read the header line-by-line and remove the hash if it's there. We only need to support the case where it is the first character I guess

**Comment by dhubber** [comment]
Yes, it should just be the first character so it should not be too difficult hopefully.
